test/end-to-end/test_pkg_download.sh

```diff
@@ -108,24 +110,41 @@ success_from_command_line() {
 }
 
 success_from_file() {
-    
     before_test
 
-    echo "==========Testing file idents"
-    
+    echo "--- Testing file idents"
+
     echo "core/gzip" > ${IDENT_FILE}
-    CMD="$HAB pkg download --download-directory=${CACHE_DIR} --file=${IDENT_FILE}"   
+    CMD="$HAB pkg download --download-directory=${CACHE_DIR} --file=${IDENT_FILE}"
```

Comment:
When running these locally with the [e2e_local.sh](https://github.com/habitat-sh/habitat/blob/559d8dcef2bfd2ac85c1aca464adaeecd0599e6d/e2e_local.sh) script, I get failures because of the `HAB_BLDR_CHANNEL` environment being set to `dev`, because all the packages that are being downloaded aren't present in that channel.

Long term, I'm not sure what the best pattern for this should be, since we still want to ensure that all the packages being built in this pipeline are properly resolved. Adding `--channel=stable` to these download commands would do the trick, as would `unset HAB_BLDR_CHANNEL` before executing the test functions.

Reply:
Ahh, I wasn't aware of that script. Thank you!

I've added the explicit `--channel stable` [here](https://github.com/habitat-sh/habitat/pull/7104/commits/6c8b27a8f8dcb2c3b80b31f1ee7dbe19ce898d25) and `unset HAB_BLDR_CHANNEL` [here](https://github.com/habitat-sh/habitat/pull/7104/commits/12273bbfb71209796afeb1935a06ca7e845d6819)